TCAPracticeCore/Sources/TCAPracticeCore/QuizFeatureReducer.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  QuizFeatureReducer.swift
+//  TCAPractice
+//
+//  Created by S J on 12/22/23.
+//
+
+//keep business logic away from views, that's why I put it in TCAPracticeCore(platform independent)
+import Foundation
+import ComposableArchitecture
+
+public enum SwipeType {
+  case left
+  case right
+}
+
+@Reducer
+public struct QuizFeatureReducer {
```

Comment:
When you swipe, the question remains on screen and remains intractable during the sleep. If you want to fix that in this PR you can. Otherwise I'll open that up as a ticket for someone else to tackle.